1323.maximum-69-number.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * @param {number} num
+ * @return {number}
+ */
+
+/* URL of this problem
+    https://leetcode.com/problems/maximum-69-number/
+*/
+
+var maximum69Number  = function(num) {
+    // convert num to array
```

Comment:
It's nice to have a comment. but it doesn't make sense. number cannot convert to array.